src/ahbicht/__init__.py

```diff
@@ -1,3 +1,25 @@
 """
 AHBicht is a lark based parser and evaluation framework for conditions that occur in Anwendungshandbüchern (AHB).
 """
+import sys
+from enum import Enum
+
+if sys.version_info.major == 3 and sys.version_info.minor >= 11:
+    from enum import StrEnum as std_lib_strenum
```

Comment:
Würde es nicht als alias importieren
```suggestion
    from enum import StrEnum
```

Reply:
https://github.com/Hochfrequenz/ahbicht/pull/247/commits/55f0b7dd1d4c5cd395208ebce08ef1334e63096c